ethanschreur/kids-krafts
# Sellers can filter orders

**What does this PR do?**
- This pull request allows sellers to filter orders

**Description of the task to be completed** 
- Add a filter option in the orders section for the following columns:
  - Name (alphabetical)
  - Email (alphabetical)
  - Pickup Time (value search)
  - Order Status (dropdown)
  - Payment Type (dropdown)
  - Payment Status (dropdown)

**How should this  be tested**
- (On Windows) Run `. venv/scripts/activate` 
- `python run.py`
- Go to http://127.0.0.1:5000/orders (log in if you have to)
- Fill out the "Add Order" form and enter orders with a variety of info so you can test the filters
- Try out the filters and confirm that it works correctly
